### PR TITLE
Render the date of a news item on its detail view.

### PR DESCRIPTION
**Pull request by mbaechtold**
screenshot

### CODE REVIEW

**Comment by jone** [comment]
could you add a screenshot?

**Comment by mbaechtold** [comment]
@jone, I've added a screenshot.

File: ftw/news/viewlets/news_date.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class NewsDateViewlet(ViewletBase):
+
+    template = ViewPageTemplateFile('news_date.pt')
+
+    def render(self):
+        return self.template()
+
+    def news_date(self):
```
Comment (jone) [inline ftw/news/viewlets/news_date.py]: The template is calling this method twice (`tal:condition` and `tal:content`), which takes unnecessary time and can be optimized.
I'd move calculate the date in an `update` method (which should automatically be called by the viewlet manager) and store the news-date as attribute on self.

Comment (mbaechtold) [inline ftw/news/viewlets/news_date.py]: Thank you for the nice explanation :+1:

**Pull request merged**